draft-ietf-scone-protocol.md

```diff
@@ -226,6 +226,22 @@ tuple for the duration of that flow.  For rate limiting networks, rate limiting
 policies often apply on the level of a device or subscription, but endpoints
 cannot assume that this is the case.  A separate signal can be sent for each flow.
 
+## Following Advice
+
+Applications that receive throughput advice can follow that advice in the way
+that best suits the needs of the application.
```

Comment:
```suggestion
The TRONE signal is advisory (see {{advisory-signal}}).
Applications that chose to follow it will do so in the way that best suits their needs.
```